test/hotspot/jtreg/vmTestbase/nsk/jvmti/GetClassFields/getclfld007/TestDescription.java

```diff
@@ -31,9 +31,9 @@
  * DESCRIPTION
  *     The test exercises JVMTI function
  *         GetClassFields(clazz, fieldCountPtr, fieldsPtr).
- *     The test checks if the function returns the expected list of fields.
- *     That is the field list contains only directly declared (not inherited)
- *     fields.
+ *     The test checks if the function returns the expected list of fields:
+ *         - the list contains only directly declared (not inherited) fields;
+ *         - fields are returned in the order they occur in the class file.
```

Comment:
Is the order of the fields in the class file guaranteed to be the same as the declaration order?

Reply:
Good question.
Javac works this way, but looks like this is not a requirement.
Looks like the test should use some class file parser to get actual field order

Reply:
Updated the test to get expected field order (as well as field name and type) from class file